inst/skeletons/example_report.Rmd

```diff
@@ -0,0 +1,35 @@
+---
+title: "Example Report"
+author: "Joe Bloggs"
+params:
+  grouped_plot: TRUE
+output:
+  html_document: default
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, collapse = TRUE, fig.width = 8, fig.height = 6, dpi = 150)
```

Comment:
Petty, but this makes for large figures to start with.`dpi = 70` should be good enough

Reply:
closed with https://github.com/reconhub/reportfactory/pull/68/commits/e60198f05faaf899a33aab96f3d9634ab3413252